### DISCUSSION TITLE
player regen

### DISCUSSION THREAD

**Discussion by Airomis** [eccentricdevotion/TARDIS#889] [Ideas]
### Describe the feature request

i noticed that there has been some work involving regin. could this be powered by the atron the player has. server config for the amount needed defult say 1k to 100 % 100 atron to a regin to 10% food and health. this would give more use to atron energy the timelord has and a penilty if they " die " as it could slow down their unlocks 

### Describe alternatives you've considered

nothing just waiting to see what the new system is going to be and how it works

**Comment by eccentricdevotion** [comment]
Added to the todo list
